pootle/core/forms.py

```diff
@@ -190,7 +190,8 @@ class FormWithActionsMixin(forms.Form):
     select_all = forms.BooleanField(
         required=False,
         label=_(
-            "Select all matching filter criteria, including those not shown"),
+            "Select all items matching filter criteria, including those not "
+            "shown"),
```

Comment:
To be honest didn't check if this was about suggestions or units. It looked like a base form class. Will check.